staging_docs/user/guides/06-sign-packages.md

```diff
@@ -0,0 +1,49 @@
+# Sign RPM Packages
+
+Sign an RPM Package using a registered RPM signing service.
+
+Currently, only on-upload signing is supported.
+
+## On Upload
+
+!!! tip "New in 3.26.0 (Tech Preview)"
+
+Sign an RPM Package when uploading it to a Repository.
+
+### Pre-requisites
+
+- Have an `RpmPackageSigningService` registered
+  (see [here](site:pulp_rpm/docs/admin/guides/add-package-signing-service/#package-signing)).
+- Have the V4 fingerprint of the key you want to use. The key should be acessible by the SigningService you are using.
+
+### Instructions
+
+1. Create or update a Repository using both `package_signing_service` and `package_signing_fingerprint` params.
+2. Upload a Package with the `sign_package` param set to `True`.
+
+### Example
+
+```bash
+# Create a Repository w/ required porams
+http POST $API_ROOT/repositories/rpm/rpm \
+  name="MyRepo" \
+  package_signing_service=$SIGNING_SERIVCE_HREF \
+  package_signing_fingerprint=$SIGNING_FINGERPRINT \
+
+# Upload a package w/ signing enabled
+pulp rpm content upload \
+  --repository ${REPOSITORY} \
+  --file ${FILE} \
+  --sign-package True
```

Comment:
Do I read this correctly that one need to specify on upload per package whether to sign it or not? Meaning that in repo there could be mixture of signed/unsigned content?
Have we considered to automatically sign on upload if the package signing service was defined on the repo?In the similar fashion as we do with metadata signing service?

Reply:
Yes, you understand correctly.
I remember we have discussed the possibility of doing something similar to this, I'm not sure why I didn't commit to this design.

I'll do like that, really looks more consistent with the usecase we want to support.